core/src/dird/dird.cc

```diff
@@ -469,7 +469,12 @@ int main(int argc, char* argv[])
 
   //   InitDeviceResources();
 
-  StartStatisticsThread();
+  if (!StartStatisticsThread()) {
+    BErrNo be;
+    Emsg1(M_WARNING, 0,
```

Comment:
```suggestion
    Emsg1(M_ERROR_TERM, 0,
```

Reply:
This way, the daemons will always shutdown, since the default behavior is now to not start.
I changed the message to be displayed from within the `StartStatisticsThread()`